<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>115</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sans-serif</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
font: 12pt &quot;sans-serif&quot;;
background: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
font: 24pt &quot;sans-serif&quot;;
color: #fff;</string>
   </property>
   <property name="text">
    <string>Daily Task Planner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>sans-serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 14pt &quot;sans-serif&quot;;
	color: #000;
	border-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Designed by Tushar Basak</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>115</y>
     <width>241</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;sans-serif&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;
padding-left: 10px;
background-color: #fff;
border: 2px solid  qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;sans-serif&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;
background-color: #fff;
border: 2px solid  qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>341</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
font: 12pt &quot;sans-serif&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="min_btn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>0</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/min.svg</normaloff>assets/min.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>0</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/close.svg</normaloff>assets/close.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for header area  ####################################################*/

QCalendarWidget QWidget {
  alternate-background-color: #b8e2ff;
}

/* style for top navigation area ###############################################*/

#qt_calendar_navigationbar {
  background-color: #fff;
  border: 2px solid qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
  border-bottom: 0px;
  border-top-left-radius: 5px;
  border-top-right-radius: 5px;
}

/* style for month change buttons ############################################ */

#qt_calendar_prevmonth,
#qt_calendar_nextmonth {
  /* border delete */
  border: none;
  /* delete default icons */
  qproperty-icon: none;

  min-width: 13px;
  max-width: 13px;
  min-height: 13px;
  max-height: 13px;

  border-radius: 5px;
  /* set background transparent */
  background-color: transparent;
  padding: 5px;
}

/* style for pre month button ############################################ */

#qt_calendar_prevmonth {
  /* set text for button */
  /*qproperty-text: &quot;&gt;&quot;;*/
  margin-left: 5px;
  image: url(:/icon/arrow-119-48.ico);
}

/* style for next month button ########################################### */
#qt_calendar_nextmonth {
  margin-right: 5px;
  image: url(:/icon/arrow-19-48.ico);
  /* qproperty-text: &quot;&gt;&quot;; */
}
#qt_calendar_prevmonth:hover,
#qt_calendar_nextmonth:hover {
  background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
}

#qt_calendar_prevmonth:pressed,
#qt_calendar_nextmonth:pressed {
  background-color: rgba(235, 235, 235, 100);
}

/* Style for month and yeat buttons #################################### */

#qt_calendar_yearbutton {
  color: #000;
  margin: 5px;
  border-radius: 5px;
  font-size: 13px;
  padding: 0px 10px;
}

#qt_calendar_monthbutton {
  width: 110px;
  color: #000;
  font-size: 13px;
  margin: 5px 0px;
  border-radius: 5px;
  padding: 0px 2px;
}

#qt_calendar_yearbutton:hover,
#qt_calendar_monthbutton:hover {
  background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
}

#qt_calendar_yearbutton:pressed,
#qt_calendar_monthbutton:pressed {
  background-color: rgba(235, 235, 235, 100);
}

/* Style for year input lineEdit ######################################*/

#qt_calendar_yearedit {
  min-width: 53px;
  color: #000;
  background: transparent;
  font-size: 13px;
}

/* Style for year change buttons ######################################*/

#qt_calendar_yearedit::up-button {
  image: url(:/icon/arrow-151-48.ico);
  subcontrol-position: right;
}

#qt_calendar_yearedit::down-button {
  image: url(:/icon/arrow-213-48.ico);
  subcontrol-position: left;
}

#qt_calendar_yearedit::down-button,
#qt_calendar_yearedit::up-button {
  width: 10px;
  padding: 0px 5px;
  border-radius: 3px;
}

#qt_calendar_yearedit::down-button:hover,
#qt_calendar_yearedit::up-button:hover {
  background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
}

/* Style for month select menu ##################################### */

#calendarWidget QToolButton QMenu {
  background-color: white;
}
#calendarWidget QToolButton QMenu::item {
  /*padding: 10px;*/
}
#calendarWidget QToolButton QMenu::item:selected:enabled {
  background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
}

#calendarWidget QToolButton::menu-indicator {
  image: none;
  nosubcontrol-origin: margin;
  subcontrol-position: right center;
  margin-top: 10px;
  width: 20px;
}

/* Style for calendar table ########################################## */
#qt_calendar_calendarview {
  /* Remove the selected dashed box */
  outline: 0px;

  border: 2px solid qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgba(0, 172, 238, 1), stop: 1 rgba(2, 126, 251, 1));
  border-top: 0px;
  border-bottom-left-radius: 5px;
  border-bottom-right-radius: 5px;
}

#qt_calendar_calendarview::item:hover {
  border-radius: 5px;
  background-color: #aaffff;
}

#qt_calendar_calendarview::item:selected {
  background-color: #55aa7f;
  border-radius: 5px;
}</string>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="assets/resource.qrc"/>
 </resources>
 <connections/>
</ui>
